src/components/appSwitch/AppSwitch.js

```diff
@@ -105,6 +105,9 @@ const AppSwitch = (props: LoadingProps) => {
     let { path, params } = destinationPath
     if (path || params) {
       path = path || 'AppNavigation/Dashboard/Home'
+      if (params && (params.paymentCode || params.code)) {
```

Comment:
does all the links are pointing to the root ? if not - we need to store the initial route from the url and redirect to it after the link was handled

Reply:
Yes, all the link are pointing to the root